Makefile

```diff
@@ -196,9 +198,9 @@ generate: ## Generate code
 .PHONY: generate-go
 generate-go: $(CONTROLLER_GEN) $(MOCKGEN) $(CONVERSION_GEN) $(KUBEBUILDER) $(KUSTOMIZE) ## Runs Go related generate targets
 	go generate ./...
```

Comment:
this might also need to be done in the api folder

Reply:
Thanks for review, it is taken care of already by #222.